src/desktop/analytics/main_layout.js

```diff
@@ -91,7 +91,7 @@ class PageTimeTracker {
     this.timer = setTimeout(() => {
       let trackingOptions = {}
 
-      const referrer = window.analytics.__artsyReferrer
+      const referrer = window.analytics.__artsyClientSideRoutingReferrer
```

Comment:
Only change in this file (prettier / eslint churn) -- updated this variable name to be more descriptive